docs/_layouts/default.html

```diff
@@ -1,20 +1,15 @@
 <!DOCTYPE html>
-<html>
-
-  {% include head.html %}
-
-  <body>
-
-    {% include header.html %}
-
-    <div class="page-content">
-      <div class="wrapper">
-        {{ content }}
-      </div>
+<html lang="en">
+{% include head.html %}
+<body>
+  {% include header.html %}
+
+  <div class="content">
+    <div class="container container-2">
```

Comment:
This might be a bit of a nitpick, but `container-2` does not tell me much in the way of "what separates this from the normal container". Might choose a slightly more telling name?